docs/src/main/asciidoc/cdi.adoc

```diff
@@ -100,8 +100,8 @@ public class Translator {
 == _Q: Can I use setter and constructor injection?_
 
 A: Yes, you can.
-In fact, in CDI the "setter injection" is superseded by more powerful https://docs.jboss.org/cdi/spec/2.0/cdi-spec.html#initializer_methods[initializer methods, window="_blank"].
-Intializers may accept multiple parameters and don't have to follow the JavaBean naming conventions.
+In fact, in CDI the "setter injection" is superseeded by more powerful https://docs.jboss.org/cdi/spec/2.0/cdi-spec.html#initializer_methods[initializer methods, window="_blank"].
```

Comment:
This change is incorrect.

Reply:
You are right, I've reverted that change.